tests/python/gaia-ui-tests/gaiatest/apps/homescreen/app.py

```diff
@@ -115,10 +115,10 @@ def number_of_columns(self):
 
     def delete_app(self, app_manifest):
         app = self.installed_app(app_manifest)
-        remove_action_element = self.marionette.find_element(*self._remove_locator)
-        app.move_to(remove_action_element)
-        from gaiatest.apps.homescreen.regions.confirm_dialog import ConfirmDialog
-        return ConfirmDialog(self.marionette)
+        app.long_tap()
+
+        from gaiatest.apps.homescreen.regions.bottom_bar import BottomBar
+        return BottomBar(self.marionette)
```

Comment:
I think you need to do the BottomBar(self.marionette).tap_remove() here, because that's what used to be part of delete_app here and I think it's part of the delete_app action.